byte NONE = 0
byte OBJECT_ATTACHED = 1
byte OBJECT_DETACHED = 2
byte object_state_change

string[] joint_names
action_dispatcher/JointTrajectoryPositions[] positions
